lib/marc_format_processor.rb

```diff
@@ -120,7 +120,7 @@ def government_document?(record)
   # Check if 260b OR 264b contain variations of "University Press"
   def university_press?(record)
     field_260b_264b = Traject::MarcExtractor.cached('260b:264b', separator: nil).extract(record)
-    field_260b_264b.grep(/University Press/i).any?
+    field_260b_264b.grep(/University|Press/i).any?
```

Comment:
This is saying match anything that has University OR Press. Isn't that too permissive? The following would be matches:

```
University Press
University Very Special Press
Press
Pressers
University Machine
University Press
University Very Special Press
Press
Pressers
University Machine
Impressive Stuff
```

I think it'd be better to do `/University\b.*\bpress\b/i`, see https://rubular.com/r/YlKC4rvzpQwRPk

Reply:
good point! will change, thanks